boards/BMS/Src/App/App_TractiveSystem.c

```diff
@@ -48,7 +48,7 @@ float App_TractiveSystem_GetCurrent(struct TractiveSystem *ts)
     float low_res_current  = App_TractiveSystem_GetLowResCurrent(ts);
     float high_res_current = App_TractiveSystem_GetHighResCurrent(ts);
 
-    if (low_res_current < HIGH_RES_MAX_CURRENT_READING)
+    if (low_res_current < HIGH_RES_MAX_CURRENT_READING && low_res_current > -HIGH_RES_MAX_CURRENT_READING)
```

Comment:
I'm not too sure what's going on here. Under different circumstances, can the ts return negative values for current?

Reply:
Yes. Depending on charging vs discharging